vaadin-multi-select-list-mixin.html

```diff
@@ -33,6 +33,7 @@
         /**
          * Array of indexes of the items selected in the items array
          * Note: Not updated when used in single selection mode.
+         * @type {Array<string> | null | undefined}
```

Comment:
Actually this could be shortened to `@type {string[] | undefined}` and the result would still be the same including `| null`, but maybe it's better to list it explicitly.

Reply:
Thanks, updated as you suggested.